libraries/botframework-connector/src/auth/skillValidation.ts

```diff
@@ -136,6 +136,13 @@ export namespace SkillValidation {
             ? GovernmentConstants.ToBotFromEmulatorOpenIdMetadataUrl
             : AuthenticationConstants.ToBotFromEmulatorOpenIdMetadataUrl;
 
+        // Add allowed token issuers from configuration (if present)
+        if (authConfig.validTokenIssuers?.length > 0) {
+            ToBotFromBotOrEmulatorTokenValidationParameters.issuer = [
+                ...ToBotFromBotOrEmulatorTokenValidationParameters.issuer,
+                ...this.authConfiguration.validTokenIssuers,
```

Comment:
It seems like this list will continue to grow as long as a bot runs since we're pushing entries into a static array. Is that a concern?

Reply:
We updated how the issuers were assigned, we defined a shallow clone object assigning new `issuers` without altering the `ToBotFromBotOrEmulatorTokenValidationParameters` constant object.